freetextresponse/private/edit.html

```diff
@@ -0,0 +1,174 @@
+<div class="wrapper-comp-settings is-active editor-with-buttons xblockfreetext_edit" id="settings-tab">
+  <ul class="list-input settings-list">
+    <!-- TODO: Add an entry here for each field -->
```

Comment:
Actually, see note above about using `xblock-utils` instead here.. We can discuss further.